gocov/convert/convert.go

```diff
@@ -57,9 +54,42 @@ func ConvertProfiles(filenames ...string) ([]byte, error) {
 		if err != nil {
 			return nil, err
 		}
-		for _, p := range profiles {
-			if err := converter.convertProfile(packages, p); err != nil {
-				return nil, err
+
+		mapUniqPackageNames := make(map[string]interface{})
+		uniqPackageNames := make([]string, 0, len(profiles))
+		for _, profile := range profiles {
+			packageName := filepath.Dir(profile.FileName)
```

Comment:
```suggestion
			packageName := path.Dir(profile.FileName)
```

Reply:
totally agree with you, i thought that i changed it. Fixed